packages/ember-views/lib/views/states/has_element.js

```diff
@@ -15,8 +16,21 @@ var hasElement = Object.create(_default);
 
 assign(hasElement, {
   $(view, sel) {
-    var elem = view.element;
-    return sel ? jQuery(sel, elem) : jQuery(elem);
+    if (isEnabled('ember-views-tagless-jquery') && view.tagName === '') {
```

Comment:
I need to confirm with @mmun that this is handled correctly by the feature flag Babel plugin. I believe it is, I just want to make sure.